private/bufpkg/buflock/buflock.go

```diff
@@ -152,10 +152,3 @@ func ExternalConfigDependencyV1Beta1ForDependency(dep Dependency) ExternalConfig
 type ExternalConfigVersion struct {
 	Version string `json:"version,omitempty" yaml:"version,omitempty"`
 }
-
-// isValidDigest returns true when the digest string is successfully parsed
-// by the `manifest` pkg. Older buf digests are not considered valid (b1/b3).
-func isValidDigest(digest string) bool {
```

Comment:
This was too lenient before - it would also ignore a digest field in the newer format that wasn't valid (wrong encoding, length, algorithm, etc.). We only wanted to ignore buf.lock files which may have potentially contained a b1/b3 digest value.